federation/pkg/kubefed/init/init.go

```diff
@@ -240,6 +241,20 @@ func initFederation(cmdOut io.Writer, config util.AdminConfig, cmd *cobra.Comman
 		if err != nil {
 			return err
 		}
+	}
+
+	//as the federation control plane is up and responding
+	//we create a config map in the fed control plane
+	//which subsequently is consumed by the dns servers of the yet to be registered clusters
+	//expecting that when ever a new cluster registers with it
+	//the control plane will create a config map in that cluster
```

Comment:
yes I did test it on a federation with 1 cluster.
I will retest it again anyways, after fixing other comments.